WooCommerce/UITestsFoundation/Screens/Login/HelpScreen.swift

```diff
@@ -0,0 +1,84 @@
+import ScreenObject
+import XCTest
+
+public final class HelpScreen: ScreenObject {
+
+    private let contactSupportButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Contact Support"]
+    }
+
+    private let emailTextFieldGetter: (XCUIApplication) -> XCUIElement = {
+        $0.textFields["Email"]
+    }
+
+    private let helpListFirstOptionGetter: (XCUIApplication) -> XCUIElement = {
+        $0.scrollViews.element.staticTexts.element(boundBy: 1)
+    }
+
+    private let helpNavigationBarGetter: (XCUIApplication) -> XCUIElement = {
+        $0.navigationBars["Help"]
+    }
+
+    private let messageTextFieldGetter: (XCUIApplication) -> XCUIElement = {
+        $0.scrollViews.element.textViews.element(boundBy: 0)
+    }
+
+    private let okButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["OK"]
+    }
+
+    private let subjectTextFieldGetter: (XCUIApplication) -> XCUIElement = {
+        $0.scrollViews.element.textFields.element(boundBy: 0)
+    }
+
+    private let submitButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Submit Support Request"]
+    }
+
+    private var contactSupportButton: XCUIElement { contactSupportButtonGetter(app) }
+    private var emailTextField: XCUIElement { emailTextFieldGetter(app) }
+    private var helpListFirstOption: XCUIElement { helpListFirstOptionGetter(app) }
+    private var helpNavigationBar: XCUIElement { helpNavigationBarGetter(app) }
+    private var messageTextField: XCUIElement { messageTextFieldGetter(app) }
+    private var okButton: XCUIElement { okButtonGetter(app) }
+    private var subjectTextField: XCUIElement { subjectTextFieldGetter(app) }
+    private var submitButton: XCUIElement { submitButtonGetter(app) }
```

Comment:
One could argue that "Help" screen and "Contact Support" screen are different screens, but I agree that it makes sense to keep them combined in one class, for the moment at least. I'd suggest reordering the elements declarations and adding a few comments. Currently, they are mixed, and it's not super intuitive what is what (for example, it took me some time to realize what `helpListFirstOption` is)ю

```suggestion
    // "Help" screen elements
    private let helpNavigationBarGetter: (XCUIApplication) -> XCUIElement = {
        $0.navigationBars["Help"]
    }

    private let contactSupportButtonGetter: (XCUIApplication) -> XCUIElement = {
        $0.staticTexts["Contact Support"]
    }

    private let emailTextFieldGetter: (XCUIApplication) -> XCUIElement = {
        $0.textFields["Email"]
    }

    private let okButtonGetter: (XCUIApplication) -> XCUIElement = {
        $0.buttons["OK"]
    }

    // "Contact Support" screen elements
    private let helpListFirstOptionGetter: (XCUIApplication) -> XCUIElement = {
        $0.scrollViews.element.staticTexts.element(boundBy: 1)
    }

    private let subjectTextFieldGetter: (XCUIApplication) -> XCUIElement = {
        $0.scrollViews.element.textFields.element(boundBy: 0)
    }

    private let messageTextFieldGetter: (XCUIApplication) -> XCUIElement = {
        $0.scrollViews.element.textViews.element(boundBy: 0)
    }

    private let submitButtonGetter: (XCUIApplication) -> XCUIElement = {
        $0.buttons["Submit Support Request"]
    }

    private var helpNavigationBar: XCUIElement { helpNavigationBarGetter(app) }
    private var contactSupportButton: XCUIElement { contactSupportButtonGetter(app) }
    private var emailTextField: XCUIElement { emailTextFieldGetter(app) }
    private var okButton: XCUIElement { okButtonGetter(app) }

    private var helpListFirstOption: XCUIElement { helpListFirstOptionGetter(app) }
    private var subjectTextField: XCUIElement { subjectTextFieldGetter(app) }
    private var messageTextField: XCUIElement { messageTextFieldGetter(app) }
    private var submitButton: XCUIElement { submitButtonGetter(app) }
```

(just in case, the suggestion code is working, if you decide to go for it)

Reply:
thanks for the suggestion! yeah it does make it clearer, i made the update in https://github.com/woocommerce/woocommerce-ios/pull/9464/commits/6beb89818c5788fdfc448703298595c0f378d276 

and yeah `helpListFirstOption` probably isn't the best name but once you go through the test it would make sense. was trying really hard to find a good name and that's the best i could come up with 😅